src/drogon_http_server.cpp

```diff
@@ -0,0 +1,133 @@
+//*****************************************************************************
+// Copyright 2024 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+#include "drogon_http_server.hpp"
+
+#include <chrono>
+#include <limits>
+#include <thread>
+#include <utility>
+
+#include <drogon/drogon.h>
+
+#include "logging.hpp"
+#include "mediapipe/framework/port/threadpool.h"
+
+namespace ovms {
+
+static const int THREAD_COUNT_FOR_DROGON_LISTENER = 3;  // TODO: how many is best perf?
```

Comment:
Just add separate task to fine tune/optimize with drogon?

Reply:
done